src/platform/api/pythonApiTypes.ts

```diff
@@ -145,7 +145,7 @@ export type Environment = EnvironmentPath & {
     /**
      * Carries Python version information known at this moment.
      */
-    readonly version: VersionInfo & {
+    readonly version?: VersionInfo & {
```

Comment:
Bug, need to try to move to the published API types, but blocked due to upstream issue for now